docroot/themes/humsci/humsci_basic/src/js/tables/wrap.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Wrap every table in a class that will allow us to create more responsive styling
```

Comment:
responsive styling? isn't it more about accessibility?

Reply:
I guess it's about accessibility in a way, but accessibility to all the table data on small screens. I'm not aware of any other accessibility benefit.